tests/utils/test_utils.py

```diff
@@ -33,18 +75,29 @@ def test_modify():
     mono.update_specs(config.dynamics.quasistatic_bulk_spec(),
                       reset=True)
     modifiers = {
-        'apical' : {
+        'apical': {
             'edge': {'line_tension': 1.},
             'face': {'contractility': 0.2},
-            },
-        'basal' : {
+        },
+        'basal': {
             'edge': {'line_tension': 3.},
             'face': {'contractility': 0.1},
-            }
         }
+    }
 
     utils.modify_segments(mono, modifiers)
     assert mono.edge_df.loc[mono.apical_edges,
                             'line_tension'].unique()[0] == 1.
     assert mono.edge_df.loc[mono.basal_edges,
                             'line_tension'].unique()[0] == 3.
+
+
+def test_ar_calculation():
+    sheet = Sheet('test', *three_faces_sheet())
+    e0 = sheet.edge_df.index[0]
+    face = sheet.edge_df.loc[e0, 'face']
+    sheet.edge_df = sheet.edge_df.loc[sheet.edge_df.index[1:]].copy()
+    close_face(sheet, face)
+
+    sheet.face_df['AR'] = utils.ar_calculation(sheet)
+    assert 'AR' in sheet.face_df
```

Comment:
Maybe a better test here would be to deform the input sheet and asserting that the aspect ratio is coherent

Initialy, the three hexagonal sheets have an aspect ratio of (I think) $\sqrt{3}$. If you do something like `sheet.vert_df['x'] *= 2`, the ar will change, you can test that both before and after the change, the ar is consistent. In practice, you can compute it in a notebook to get the values before writing the test..